pages/having-a-child.js

```diff
@@ -23,14 +24,101 @@ export default function lifejourney() {
       <div id="wb-bc" className="container pt-4">
         <Breadcrumb />
       </div>
-      <Accordion>
-        <div className="pr-6">
-          <MoreInfo text={t.moreInfoCommunity} />
-          <MoreInfo text={t.moreInfoFertility} />
-          <MoreInfo text={t.moreInfoHealth} />
-          <MoreInfo text={t.moreInfoFinancial} />
+
+      <div className="container">
+        <div className="w-3/12 sticky top-0">
+          <SideMenu />
+        </div>
+
+        <div className="w-9/12 float-right">
+          <Accordion
+            id="GettingReady"
+            title="Getting Ready"
+            summary="Getting Ready summary text"
+          >
```

Comment:
I've noticed the accordions all have an `id="accordion"`, which we should remove. IDs are supposed to be unique on the page, so better to not have one.

Reply:
done already, the id on the accordion becomes the anchor for the side menu links